physionet-django/notification/templates/notification/email/notify_aws_access_request.html

```diff
@@ -0,0 +1,11 @@
+Dear {{ user.get_full_name }},
+
+You have request access to use {{ project }} in AWS S3 Storage.
```

Comment:
Change `You have request access to use {{ project }} in AWS S3 Storage.`
to `You have requested access to use {{ project }} in AWS S3 Storage.`